test/run_test.py

```diff
@@ -36,7 +37,8 @@
     'distributed/test_data_parallel',
     'distributed/test_distributed_fork',
     'distributed/test_distributed_spawn',
-    'test_distributions',
+    'distributions/test_constraints.py',
+    'distributions/test_distributions',
```

Comment:
is there a reason some files end with .py and others don't?

Reply:
Sorry, that was a typo, will fix. The test should still run with or without .py.